examples/pylint/W0511_fixme.py

```diff
@@ -0,0 +1,3 @@
+def fishyFunction() -> None:
```

Comment:
Make sure this is snake case (and there are other places too). You could keep the `FIXME` but follow it with a comment like "implement this function", and remove the `print` call.